src/pages/ReportAvatar.tsx

```diff
@@ -23,21 +23,38 @@ type ReportAvatarProps = ReportAvatarOnyxProps & StackScreenProps<AuthScreensPar
 function ReportAvatar({report = {} as Report, route, policies, isLoadingApp = true}: ReportAvatarProps) {
     const policyID = route.params.policyID ?? '-1';
     const policy = policies?.[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`];
-    const policyName = ReportUtils.getPolicyName(report, false, policy);
-    const avatarURL = ReportUtils.getWorkspaceAvatar(report);
+
+    const attachment = useMemo(() => {
+        if (ReportUtils.isGroupChat(report) && !ReportUtils.isThread(report)) {
+            return {
+                source: report?.avatarUrl ? UserUtils.getFullSizeAvatar(report.avatarUrl, 0) : ReportUtils.getDefaultGroupAvatar(report?.reportID ?? ''),
```

Comment:
`report?.avatarUrl` should be enough as you are nut supposed to be able to access this page if report does not have a custom avatar